testing/test.js

```diff
@@ -0,0 +1,10 @@
+const puppeteer = require('puppeteer');
+const pageUrl = 'http://localhost:3418/';
+
+describe('ensure testing config', () => {
+
+  test('jest config', () => {
+    const message = 'test run';
+    expect(message).toEqual('test run');
+  });
+});
```

Comment:
over all great work Daniel, your code looks clean and simple to follow!!

Reply:
Thanks Kevin